content/blog/setting-up-harbor-as-a-local-container-registry-in-hpe-private-cloud-ai.md

```diff
@@ -0,0 +1,299 @@
+---
+title: Setting up Harbor as a local container registry in HPE Private Cloud AI
+date: 2025-07-03T07:21:44.007Z
+author: Guoping Jia
+authorimage: /img/guoping.png
+disable: false
+tags:
```

Comment:
Hi Guoping,
Could you please one additional tag:
hpe-private-cloud-ai

FYI - We have a landing page for HPE PCAI in the HPE Dev Community portal here:
https://developer.hpe.com/platform/hpe-private-cloud-ai/home/